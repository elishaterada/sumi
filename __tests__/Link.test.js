import React from 'react';
import renderer from 'react-test-renderer';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';

import LinkClass from '../src/components/Link';

expect.addSnapshotSerializer(createSerializer(emotion));

test('renders Link', () => {
  const reactComponent = renderer
    .create(<div className={LinkClass}>Test</div>)
    .toJSON();

  expect(reactComponent).toMatchSnapshot();
});
