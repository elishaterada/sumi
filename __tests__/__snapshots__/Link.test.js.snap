// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Link 1`] = `
.emotion-0 {
  font-size: 1rem;
  font-weight: 400;
  color: #2196F3;
  text-decoration: underline;
}

.emotion-0:hover,
.emotion-0:focus {
  color: #999999;
}

<div
  className="emotion-0"
>
  Test
</div>
`;
